ksp_plugin_adapter/orbit_analyser.cs

```diff
@@ -505,15 +505,78 @@ internal class OrbitAnalyser : VesselSupervisedWindowRenderer {
           field_width      : 5) {
           value = 7 * 24 * 60 * 60
       };
-  private bool autodetect_recurrence_ = true;
-  private int revolutions_per_cycle_ = 1;
-  private int days_per_cycle_ = 1;
-  private int ground_track_revolution_ = 1;
+  protected bool autodetect_recurrence_ = true;
+  protected int revolutions_per_cycle_ = 1;
+  protected int days_per_cycle_ = 1;
+  protected int ground_track_revolution_ = 1;
```

Comment:
It would be good if we avoided protected fields.  I mean, we avoid them in C++, right?

Reply:
Done.